src/services/Evernode.js

```diff
@@ -81,6 +83,15 @@ const getHosts = async (filters = null, pageSize = null, nextPageToken = null) =
     return governorClient.getHosts(filters, pageSize, nextPageToken);
 }
 
+const getCandidates = async (filters = null, pageSize = null, nextPageToken = null) => {
+    return governorClient.getCandidates(filters, pageSize, nextPageToken);
```

Comment:
As mentioned above comment, let's use `evernode.StateHelpers.getCandidateType(candidateId)` here and add candidateType property to the candidate objects.